sql-plugin/src/main/scala/com/nvidia/spark/rapids/Plugin.scala

```diff
@@ -90,6 +90,14 @@ object RapidsPluginUtils extends Logging {
       logWarning("RAPIDS Accelerator is disabled, to enable GPU " +
         s"support set `${RapidsConf.SQL_ENABLED}` to true.")
     }
+
+    if (conf.isUdfCompilerEnabled) {
+      logWarning("RAPIDS UDF compiler is enabled, to disable it " +
+      s"set `${RapidsConf.UDF_COMPILER_ENABLED}` to false.")
+    } else {
+      logWarning("RAPIDS UDF compiler is disabled, to enable it " +
```

Comment:
+1 to @jlowe's comment https://github.com/NVIDIA/spark-rapids/pull/5157#discussion_r845182574 .

Reply:
Done!